tools/lint-md/lint-md.src.mjs

```diff
@@ -17,9 +31,10 @@ const linter = unified()
 paths.forEach(async (path) => {
   const file = await read(path);
   const result = await linter.process(file);
-  if (result.messages.length) {
+  if (format) {
+    fs.writeFileSync(path, String(result));
```

Comment:
Nit: I find it clearer to call `result.toString()` directly.

Reply:
Sure, changed.